docs/documentation.md

```diff
@@ -0,0 +1,470 @@
+<!--
+   - SPDX-FileCopyrightText: 2022 Tezos Commons
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TC
+   -->
+
+# Homebase Lite
+
+
+
+
```

Comment:
One more comment about this is that we generally add pipeline steps to generate and publish this (see [tezos-btc's example](https://github.com/tz-wrapped/tezos-btc/blob/master/.buildkite/pipeline.yml#L93-L140), so including the result in the repo is unnecessary.

I think this a dedicated "publishing" issue is better suited for most of those steps (including non-doc-related actions, e.g. for a static binary), but if you want to add the most basic autodoc step that runs on every PR here that would be good.

Reply:
BTW at support of this comment, you didn't update it after the `KeyHash`->`Address` change :slightly_smiling_face: